GenerateCLP/GenerateCLP.cxx

```diff
@@ -468,6 +468,16 @@ void GeneratePre(std::ostream &sout, ModuleDescription &, int argc, char *argv[]
 }
 
 #ifdef GenerateCLP_USE_SERIALIZER
+void ReplaceCharachter(std::string& line, char character, const std::string replacement)
```

Comment:
Otherwise, you could have a function named `EscapeCharachter(std::string& text, char character)`